prepare_oim/roles/deploy_containers/pcs/templates/pcs-start.sh.j2

```diff
@@ -16,5 +16,37 @@
 systemctl start corosync
 systemctl start pacemaker
 pcs property set stonith-enabled=false
-pcs property set no-quorum-policy=ignore
+
+pcs resource create omnia_core ocf:heartbeat:podman \
+ image={{ omnia_core_image }} \
+ name={{ omnia_core_name }} \
+ reuse=true \
+ run_opts="{{ omnia_core_run_opts }}" \
+ op monitor interval={{ monitor_interval }} timeout={{ monitor_timeout }} on-fail=restart \
+ op start interval={{ start_interval }} timeout={{ start_timeout }} on-fail=restart \
+ op stop interval={{ stop_interval }}  timeout={{ stop_timeout }}
+
+pcs resource create provision ocf:heartbeat:podman \
+ name={{ provision_container_name }} \
+ image={{ provision_image }} \
+ reuse=true \
+ run_opts="{{ provision_run_opts }}"
+ op monitor interval={{ monitor_interval }} timeout={{ monitor_timeout }} on-fail=restart \
+ op start interval={{ start_interval }} timeout={{ start_timeout }} on-fail=restart \
+ op stop interval={{ stop_interval }}  timeout={{ stop_timeout }}
+
+{% if hostvars['localhost']['k8s_support'] %}
+pcs resource create kubespray ocf:heartbeat:podman \
+ name={{ kubespray_name }} \
+ image={{ kubespray_image }} \
+ reuse=true \
+ run_opts="{{ kubespray_run_opts }}"
+ op monitor interval={{ monitor_interval }} timeout={{ monitor_timeout }} on-fail=restart \
+ op start interval={{ start_interval }} timeout={{ start_timeout }} on-fail=restart \
+ op stop interval={{ stop_interval }}  timeout={{ stop_timeout }}
+{% endif %}
+
+pcs resource group add omnia_containers omnia_core provision {% if hostvars['localhost']['k8s_support'] %} kubespray {% endif %}
```

Comment:
use pcs resource name as omnia_provision and omnia_kubespray_<kubespray_version>. We will have to handle multiple kubespray container also